source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/calculation/ProjectCategory.java

```diff
@@ -20,7 +20,10 @@
 
 /**
  * The project category the calculation is for.
+ *
+ * @Deprecated To be replaced with database-backed category.
```

Comment:
Will these database-backed categories include `GO_KARTING`, so the roundtrip test file in this PR works?